### PR TITLE
#166264853 Use consola to log server run status

### PR DESCRIPTION
**Pull request by julieg18**
#### What does this PR do?
Uses consola to log server run status

#### Description of Task to be completed?
To ensure that upon a successful start of our app, our user gets a console message with `Find me on port ${port}`.

#### How should this be manually tested?
Type `npm run dev` in the terminal and you should be able to see the console message.

#### What are the relevant pivotal tracker stories?
#166264853

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "clean": "rm -rf build",
     "db:migrate": "sequelize db:migrate",
-    "db:undo:migrate": "sequelize db:migrate:undo:all"
+    "db:undo:migrate": "sequelize db:migrate:undo:all",
+    "dev": "babel-watch server.js"
```
Comment (ebenezerdon) [inline package.json]: Hi, can we move the dev script right before `test`?

Comment (julieg18) [reply package.json]: Of course, I'll get right on it!

Comment (ebenezerdon) [reply package.json]: Also, let's change your commit message from `finished` to `finishes`.
Good job!

Comment (julieg18) [reply package.json]: Commit message is changed.

**Pull request merged**